### ISSUE TITLE
Crashes Visual Studio

### ISSUE THREAD

**Issue by Wintech80** [kaifuzi/ExpTreeLib#2]
Hi I have been using your ExpList control in my app for a while now.
Everything works fine in my application and I have had it running for hours without issue.

The problem I have is if I try to open the form the control is used on design view, within seconds Visual Studio will freeze and crash with the following error logged in event log.

Faulting application name: devenv.exe, version: 17.3.32804.467, time stamp: 0x62ec695f
Faulting module name: KERNELBASE.dll, version: 10.0.22000.832, time stamp: 0x7efab3dc
Exception code: 0xc000041d
Fault offset: 0x000000000004474c
Faulting process ID: 0x6af8
Faulting application start time: 0x01d8b3698d75ec7d
Faulting application path: C:\Program Files\Microsoft Visual Studio\2022\Community\Common7\IDE\devenv.exe
Faulting module path: C:\WINDOWS\System32\KERNELBASE.dll
Report ID: b985e55c-5389-4031-a553-5d443ba24e73
Faulting package full name: 
Faulting package-relative application ID: 

I thought initially it may have been related to the stack overflow issue you mentioned but I have updated to the latest version with the same issue. I had the issue in VS 2019 and still have the issue in VS 2022

If I open the forms design code and comment out the ExpList lines I can then open and edit my form but I need to undo this everytime I build the app.

**Comment by kaifuzi** [comment]
@Wintech80 Do you have the same issue in my Demo project? For my case, I don't have the crash issue, I'm using VS2015 and VS 2019, my Windows versio is Win10 21H2. Could you please create a simple project to see do you still have the same issue?

**Comment by Wintech80** [comment]
@kaifuzi Yes, if I open just the demo project in both VS2019 and VS2022 on Windows 11 (I was having the issue on Windows 10 also). It doesn't crash if you just load the project but if you open the form in design view it will crash. Sometimes it crashes instantly sometimes after a minute or two but it's only if you open in design view.

**Comment by kaifuzi** [comment]
@Wintech80 I had same issue with you before I merged the fixing code from original author Jim Parsells. But after that, this issue never happened to me again. Since I can't replicate your issue, to be honest, I don't know how to help you.
Did you try to download the codes with zip and open the project?
